layouts/partials/deprecation-warning.html

```diff
@@ -5,7 +5,10 @@ <h3>
       {{ T "deprecation_title" }} {{ .Param "version" }}
     </h3>
     <p> Kubernetes {{ .Param "version" }} {{ T "deprecation_warning" }}
-      <a href="{{ site.Params.currentUrl }}">{{ T "latest_version" }}</a>
+     <a href="{{ site.Params.currentUrl }}">{{ T "latest_version" }}</a>
+    </p>
+    <p> {{ T "deprecation_warning_page_link" }} 
+     <a href="{{ .Permalink }}">{{ T "latest_version" }}</a>
```

Comment:
Does this automagically work for localized pages?

Reply:
Yep